### ISSUE TITLE
Invalid yield after JS minification (2)

### ISSUE THREAD

**Issue by jbress** [trullock/NUglify#301]
This code
```
let x = 1;
if (p != null && (yield isOk(p)) != null)
    x = 2;
```
Become this (NUglify 1.17.11)
```
let i = 1;
n != null && yield t(n) != null && (i = 2)
```
which is invalid

Expecting
```
let i = 1;
n != null && (yield t(n)) != null && (i = 2)
```
[1 input.js.txt](https://github.com/trullock/NUglify/files/8204227/1.input.js.txt)
[2 output actual.js.txt](https://github.com/trullock/NUglify/files/8204228/2.output.actual.js.txt)
[3 output expected.txt](https://github.com/trullock/NUglify/files/8204229/3.output.expected.txt)

**Comment by trullock** [comment]
theres probably lots of cases like this

**Comment by trullock** [comment]
Fixed in 1.17.12
